### PR TITLE
change buttons ui

### PR DESCRIPTION
**Pull request by piensaenpixel**

### CODE REVIEW

**Comment by xavijam** [comment]
Hey @piensaenpixel, why shouldn't we merge this one?

**Comment by xavijam** [comment]
We have your las CartoDB.js changes in master, time to review this PR man :)

**Comment by piensaenpixel** [comment]
@nobuti  ready :) take a look 
cc/ @xavijam 
ui-buttons

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -624,7 +640,11 @@
                             "is-my-json-valid": {
                               "version": "2.15.0",
                               "from": "is-my-json-valid@>=2.12.4 <3.0.0",
+<<<<<<< HEAD
```
Comment (nobuti) [inline npm-shrinkwrap.json]: WTF @piensaenpixel

**Comment by xavijam** [comment]
Hey, apart from the thing @nobuti has reported, I feel new buttons will collide with the table pagination buttons, try with a dataset with data and several pages.

**Comment by piensaenpixel** [comment]
Fixed all things :) take a look

buttons

**Comment by xavijam** [comment]
Hey @piensaenpixel, included the case with the table pagination??

**Comment by xavijam** [comment]
cc @matallo, because it is needed for you 💋

**Comment by matallo** [comment]
ack, @piensaenpixel already pinged me

**Pull request merged**